code/ecdsap256/Hacl.Impl.P256.LowLevel.fst

```diff
@@ -163,7 +171,66 @@ let copy_conditional out x mask =
   lemma_eq_funct_ (as_seq h1 out) (as_seq h0 out);
   lemma_eq_funct_ (as_seq h1 out) (as_seq h0 x)
 
+(*
```

Comment:
please remove this bit as it's now in another file

Reply:
Solved